static/main.css

```diff
@@ -338,6 +391,18 @@ main.normalize {
         color: #ce93d8; /* purple200 */
     }
 
+    #site-menu ul li[aria-current] a {
+        color: #90caf9; /* blue200 */
+    }
+
+    .gs-head-logo img {
+        filter: invert(87%);
+    }
+
+    .gs-head-logo a, #site-menu ul li a {
+        color: rgba(255 255 255 / 87%); /* 87% white */
```

Comment:
No need for the comment here as well, it's obvious that it's 87% white :)

Reply:
This was also done for consistency.